### DISCUSSION TITLE
Only returning black images

### DISCUSSION THREAD

**Discussion by gunnarsal** [vladmandic/sdnext#425] [Q&A] [answered]
I will set up a different discussion point as I do not find the answers from others.

I have installed vlad and it runs quickly but it only returns black images.
![image](https://user-images.githubusercontent.com/129676685/233861012-dc87468d-6db9-4624-bfba-7e6ec4225028.png)

I have used different checkpoints that work without issues with A1111, but do not work with Vlad.

The only thing that I can think of is that I have 1660 Ti and it failed to install CUDA. Is it mandatory to have it installed? If not, do I need to disable a setting that is searching for the installation?
I have the follwing CUDA settings.
![image](https://user-images.githubusercontent.com/129676685/233861046-4d8b3507-5bd7-4070-a717-d434c44dbd32.png)

What further info I can provide to get this sorted?

**Comment by ValeryEMalakhov** [comment, marked as answer]
Try the solution in [this thread](https://github.com/vladmandic/automatic/issues/345#issuecomment-1518108325). Helped me.

**Reply by gunnarsal** [reply]
Nice, found the setting I had to change... One other question though, I see the image is generated, i see it in the image browser, but i do not see it after it is generated in "results"
![image](https://user-images.githubusercontent.com/129676685/233987835-31feb7de-e5c2-46fc-9488-9544e89d5769.png)
